actions/python-setup-poetry/action.yaml

```diff
@@ -18,35 +18,28 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Set up Python ${{ inputs.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ inputs.python-version }}
-
-    - name: Get full Python version
-      id: full-python-version
-      run: echo ::set-output name=version::$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")
-      shell: bash
-
-    - name: Bootstrap poetry
+    - name: Install Poetry ${{ inputs.poetry-version }}
+      run: pipx install poetry==${{ inputs.poetry-version }}
```

Comment:
Where does `pipx` come from and does it not depend on python being installed before? Are you using a specific docker image when using the action?

Reply:
Interesting, so then the `setup-python` action below only configures python for venv with a specific version, but does not really install a system wide version like you would do locally

Reply:
Can you find a more fitting description for this action in line 2 then please?

Reply:
yep, it only takes care of our project's virtualenv and that's a good thing imo. Python is already installed by default.